tgozo19/laravel-codegen
# Add support for package auto-discovery

## Description

This pull request adds support for package auto-discovery in Laravel. With this feature, users no longer need to manually register the package's service provider in their `app/config.php` file. Instead, the package's service provider will be automatically discovered and registered by Laravel after the package is installed.

## Changes

- Added the `extra` section to the `composer.json` file to specify the package's service provider for auto-discovery
- Updated the installation instructions in the `README.md` file to reflect the changes

## How to test

To test this feature, install the package in a Laravel application and verify that its service provider is automatically registered. You can do this by running the `php artisan package:discover` command and checking that the package's service provider is listed in the output.
